.github/actions/latest-stable-tag/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "latest-stable-tag",
+  "version": "1.0.0",
+  "description": "Gets the latest stable tag in a Git repository",
+  "main": "index.js",
+  "type": "module",
+  "scripts": {
+    "test": "cross-env NODE_OPTIONS=--experimental-vm-modules npx jest"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
This is new to me!

Reply:
To me too! I have no idea what this is. Again, `npm init`.

Reply:
I removed this property, since it's not strictly necessary